templates/embed-api/server-side-authorization.html

```diff
@@ -0,0 +1,131 @@
+{% extends 'server-auth.html' %}
+
+{% block content %}
+<section>
+  <h2>Demo Overview</h2>
+  <p>{{ page.excerpt | safe }}</p>
+  <p>The charts below show usage data for the site you're visiting now, <a href="/">Google Analytics Demos & Tools</a>. Notice how you do not have to be logged in to see this data!</p>
+</section>
+
+<ul class="FlexGrid">
+  <li class="FlexGrid-item">
+    <div class="Dashboard">
+      <header class="Titles">
+        <h3 class="Titles-main">Site Traffic</h3>
+        <div class="Titles-sub">Sessions vs. Users - last 30 days</div>
+      </header>
+      <figure id="chart-1-container"></figure>
+    </div>
+  </li>
+  <li class="FlexGrid-item">
+    <div class="Dashboard">
+      <header class="Titles">
+        <h3 class="Titles-main">Most Popular Demos/Tools</h3>
+        <div class="Titles-sub">Pageviews - last 30 days</div>
+      </header>
+      <figure id="chart-2-container"></figure>
+    </div>
+  </li>
+</ul>
+
+<section>
+<h2>How it works</h2>
+
+<p>You can recreate the demo above by following the instructions and using the code below. If you want to know more, check out the <a href="https://developers.google.com/analytics/devguides/reporting/embed/">Embed API documentation</a> on our developers site.</p>
+
+<h3>Step 1: Create a service account and download the JSON key</h3>
+
+<p>Follow the steps in the Google Idendity Platform documentation to <a href="https://developers.google.com/identity/protocols/OAuth2ServiceAccount#creatinganaccount">create a service account</a> from the <a href="https://console.developers.google.com/">Google Developer Console</a>.</p>
+
+<p>Once the service account is created, you can click the <strong>Generate New JSON Key</strong> button to create and download the key and add it to your project.</p>
+
+<p class="Message"><strong>Important!</strong>&nbsp; Make sure to store your JSON key file privately and securely. Do not check it in to a public repository or store it on a public server.</p>
+
+<h3>Step 2: Add the service account as a user in Google Analytics</h3>
+
+<p>The service account you created in the previous step has an email address that you can add to any of the Google Analytics views you'd like to request data from. It's generally best to only grant the service account read-only access.</p>
+
```

Comment:
I suggest adding a tip/note about implications of adding this to an account/property/view.